### ISSUE TITLE
Code moved to a function evaluates as unequal.

### ISSUE THREAD

**Issue by viktormalik** [viktormalik/diffkemp#30]
If a part of the code is moved into a called function, it is evaluated as non-equal, since we compare function-by-function.
This happens e.g. in comparing KABI between 3.10.0-862 and 3.10.0-957, whitelist symbol `__root_device_register`. A difference is reported in functions `device_add` and `device_initialize`. Both functions contain the following code in 3.10.0-862:
```
if (!dev->device_rh)
    device_rh_alloc(dev);
```

whereas in 3.10.0-957, this is changed to:
```
device_rh_alloc(dev);
```

and the condition test is moved into `device_rh_alloc`.

**Comment by viktormalik** [comment]
This could be solved by inlining the called function in case such situation occurs.

**Comment by viktormalik** [comment]
Solved by b85d7ac575d3b0ea72f681d53886f87cf0b9cb07
